src/store/model/mediamarkt-at.ts

```diff
@@ -0,0 +1,96 @@
+import {Store} from './store';
+
+export const MediamarktAt: Store = {
+  backoffStatusCodes: [403, 429, 503],
+  currency: '€',
+  labels: {
+    captcha: {
+      container: 'p',
+      text: ['Das ging uns leider zu schnell.'],
+    },
+    maxPrice: {
+      container: 'span[font-family="price"]',
+      euroFormat: false,
+    },
+    outOfStock: [
+      {
+        container: '#root',
+        text: ['Dieser Artikel ist aktuell nicht verfügbar.'],
+      },
+      {
+        container: '#root',
+        text: ['Leider keine Lieferung möglich'],
+      },
+      {
+        container: '#root',
+        text: ['Nicht verfügbar'],
+      },
+      {
+        container: '#root',
+        text: ['Dieser Artikel ist dauerhaft ausverkauft'],
+      },
+      {
+        container: '#root',
+        text: ['Dieser Artikel ist bald wieder für Sie verfügbar'],
+      },
+    ],
+  },
+  links: [
+    {
+      brand: 'test:brand',
+      model: 'test:model',
+      series: 'test:series',
+      url: 'https://www.mediamarkt.de/de/product/-2641856.html',
+    },
+    {
+      brand: 'gainward',
+      model: 'phoenix',
+      series: '3060ti',
+      url:
+        'https://www.mediamarkt.at/de/product/_gainward-grafikkarte-geforce-rtx-3060-ti-phoenix-golden-sample-8gb-ga-3060rtx-ti-ph-gs-1815563.html',
```

Comment:
```suggestion
        'https://www.mediamarkt.at/de/product/-1815563.html',
```

Also seems we can normalize these urls by using this. Do you mind changing that?

Reply:
Done